app/javascript/src/Dashboard/components/APIDataListItem.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useRef, useEffect, useCallback } from 'react'
+
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownPosition,
+  KebabToggle,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction
+} from '@patternfly/react-core'
+
+type Props = {
+  api: {
+    id: number,
+    link: string,
+    links: Array<{
+      name: string,
+      path: string
+    }>,
+    name: string,
+    type: string,
+    updated_at: string
+  }
+}
+
+const APIDataListItem = ({ api }: Props) => {
+  const { id, name, updated_at: updatedAt, link, links } = api
+  const [isOpen, setIsOpen] = useState(false)
+  const ref = useRef(null)
+
+  const handleClickOutside = useCallback((event) => {
+    if (ref.current && !ref.current.contains(event.target)) {
+      setIsOpen(false)
+    }
+  }, [ref])
+
+  useEffect(() => {
+    if (isOpen) {
+      document.addEventListener('mousedown', handleClickOutside, false)
+    } else {
+      document.removeEventListener('mousedown', handleClickOutside, false)
+    }
+  }, [isOpen])
+
+  return (
+    <DataListItem key={id} aria-labelledby="single-action-item1">
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell key="primary content">
+              <a href={link} id="single-action-item1">
+                {name}
+              </a>
+            </DataListCell>,
+            <DataListCell key="secondary content" className="dashboard-list-secondary">
+              {updatedAt}
+            </DataListCell>
+          ]}
+        />
+        <DataListAction
+          aria-labelledby="multi-actions-item1 multi-actions-action1"
+          id="actions-menu"
+          aria-label="Actions"
+          // TODO: throwing a warning, does this prop really exist?
+          isPlainButtonAction
```

Comment:
I think we can simply remove this prop. According to PF docs is not required and I don't see any difference in the rendered output
![pf-docs](https://user-images.githubusercontent.com/13486237/95998969-b7860000-0e35-11eb-8de2-cb4221783d7a.png)
```suggestion
```

Reply:
OK sounds good to me!